src/loaders/express.js

```diff
@@ -0,0 +1,21 @@
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const helmet = require('helmet');
+const morgan = require('morgan');
+
+
+module.exports = (app) => {
+
+    // set development mode
+    if (process.env.NODE_ENV === 'development') {
+        app.use(morgan('dev'));
+    }
+    app.get('/status', (req, res) => res.status(200).send({message: "OK"}));
+
+    app.use(cors());
+
```

Comment:
I think we need Bodyparser middleware in this file

Reply:
@TalkyEk Please check **does it work on your local machine**.